src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeCustomAttributeData.cs

```diff
@@ -1126,16 +1122,13 @@ internal static bool IsAttributeDefined(RuntimeModule decoratedModule, int decor
         }
 
         private static object[] GetCustomAttributes(
-            RuntimeModule decoratedModule, int decoratedMetadataToken, int pcaCount, RuntimeType? attributeFilterType)
```

Comment:
[Every caller of this method](https://source.dot.net/#System.Private.CoreLib/src/System/Reflection/RuntimeCustomAttributeData.cs,4abc6a05a3812753,references) checks `Debug.Assert(caType is not null)` first, so this shouldn't have to be nullable.